### DISCUSSION TITLE
Configuration is not updated automatically

### DISCUSSION THREAD

**Discussion by lephuhai270** [akvorado/akvorado#1013] [Q&A]
Could Akvorado update a configuration file automatically once we change anything on any file or which container/service should we restart once update any configuration?

**Comment by vincentbernat** [comment]
You need to restart the orchestrator and then the containers impacted by your changes. Your request is tracked in #55 but it is unlikely to be implemented any time soon.

**Reply by lephuhai270** [reply]
Thanks for your help!!!
